CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [Unreleased]
+
+### Fixed
+- Derive maven compiler version globally in pom instead of setting it individually (#47)
```

Comment:
Can we agree on a standard for whether or not we link the PR to the change?

Reply:
yes for sure we can. I removed the issue reference, sticking to the current convention.